api/clock/api.go

```diff
@@ -71,7 +71,8 @@ func (a *API) Configure(message types.ClientMessage) error {
 // Data gets the current time!
 func (a *API) Data() (interface{}, error) {
 	return types.ClockResponse{
-		Time: a.time.In(a.loc()).Unix(),
+		Time:        a.time.In(a.loc()).Unix(),
+		ClockConfig: a.Config,
```

Comment:
Do we want to use different names for the config of every API, or should they all be unified under a general `Config` key?

Reply:
It doesn't matter really, as it will be flattened for the client.

Reply:
Do you mean the `a.Config`? Because I think that is unified. Otherwise, the `ClockConfig` thing is the name of the config type for that api.